lib/cli/interactive-setup/dashboard-set-org.js

```diff
@@ -13,92 +14,84 @@ const {
 
 const isValidAppName = RegExp.prototype.test.bind(/^[a-z0-9](?:[a-z0-9-]{0,126}[a-z0-9])?$/);
 
-const orgUpdateConfirm = async (inquirer, stepHistory) => {
+const orgUpdateConfirm = async (stepHistory) => {
   process.stdout.write(
     "Service has monitoring enabled, but it is configured with the 'org' to which you do not have access\n\n"
   );
-  const shouldUpdateOrg = (
-    await inquirer.prompt({
-      message: 'Would you like to update it?',
-      type: 'confirm',
-      name: 'shouldUpdateOrg',
-    })
-  ).shouldUpdateOrg;
-  stepHistory.set('shouldUpdateOrg', shouldUpdateOrg);
+  const shouldUpdateOrg = await promptWithHistory({
+    message: 'Would you like to update it?',
+    type: 'confirm',
+    name: 'shouldUpdateOrg',
+    stepHistory,
+  });
   return shouldUpdateOrg;
 };
-const appUpdateConfirm = async (inquirer, appName, orgName, stepHistory) => {
+
+const appUpdateConfirm = async (appName, orgName, stepHistory) => {
   process.stdout.write(
     "Service seems to have monitoring enabled, but configured app doesn't seem to exist in an organization.\n\n"
   );
 
-  const appUpdateType = (
-    await inquirer.prompt({
-      message: 'What would you like to do?',
-      type: 'list',
-      name: 'appUpdateType',
-      choices: [
-        { name: `Create '${appName}' app in '${orgName}' org`, value: 'create' },
-        {
-          name: 'Switch to one of the existing apps (or create new one with different name)',
-          value: 'chooseExisting',
-        },
-        { name: "Skip, I'll sort this out manually", value: 'skip' },
-      ],
-    })
-  ).appUpdateType;
-  stepHistory.set('appUpdateType', appUpdateType);
+  // TODO: FIX how the value is recorded
```

Comment:
What exactly is the problem?

Reply:
oops, that's a leftover todo - it's all good here